{{cookiecutter.project_shortname}}/invenio.cfg

```diff
@@ -132,11 +132,9 @@ APP_DEFAULT_SECURE_HEADERS['content_security_policy']['default-src'].append(
 # See https://github.com/inveniosoftware/invenio-records-resources/blob/master/invenio_records_resources/config.py
 
 # TODO: Set with your own hostname when deploying to production
-SITE_HOSTNAME = "127.0.0.1"
+SITE_UI_URL = "https://127.0.0.1"
 
-SITE_UI_URL = f"https://{SITE_HOSTNAME}:5000"
-
-SITE_API_URL = f"https://{SITE_HOSTNAME}:5000/api"
```

Comment:
I wonder why we did it that way in the first place :sweat_smile: ? Using the containerized values as defaults streamlines things.